docs/source/examples/import_classes.py

```diff
@@ -0,0 +1,60 @@
+from gpkit import Variable, Model
```

Comment:
this file needs a more descriptive name than `import_classes`, and a docstring at the very top (is it meant to be added to, or is it only for a specific example)?